sdk/spring/spring-messaging-azure-eventhubs/src/main/java/com/azure/spring/eventhubs/core/processor/DefaultEventHubsNamespaceProcessorFactory.java

```diff
@@ -49,6 +52,8 @@
     private final PropertiesSupplier<ConsumerIdentifier, ProcessorProperties> propertiesSupplier;
     private final Map<ConsumerIdentifier, EventProcessorClient> processorClientMap = new ConcurrentHashMap<>();
     private final ProcessorPropertiesParentMerger propertiesMerger = new ProcessorPropertiesParentMerger();
+    private AzureCredentialResolver<AzureTokenCredentialProvider> tokenCredentialResolver = null;
+    private DefaultAzureCredential defaultAzureCredential = null;
```

Comment:
as above. default value to be final is understandable. this is not default one? the setter below is for the same issue about this. Better to keep accordance with other places.

Reply:
Same reason.